mslice/plotting/plot_window/slice_plot.py

```diff
@@ -29,6 +29,8 @@ def __init__(self, plot_figure, canvas, slice_plotter):
         self._legend_dict = {}
         self.icut_event = [None, None]
         self.icut = None
+        pos = self.plot_figure.pos()
+        self.plot_figure.move(pos.x() - 375, pos.y())
```

Comment:
Why do we need to move the slice window as well? Shouldn't the movement width be based on the window size?

Reply:
Yes it probably should be based on the window size actually. We don't need to move the slice window, but I thought it would be better to keep them both somewhat central rather than having the slice in the centre and the cut on the right.